packages/tools/addon/templates/components/cs-version-control.hbs

```diff
@@ -34,9 +34,14 @@
   {{#liquid-if enabled rules=animationRules growDuration=250 growPixelsPerSecond=1}}
     <footer class="cs-version-control-footer cs-field-editor-section">
       <a class="text-button" {{action (perform delete) }}>Delete</a>
-      <button disabled={{eq modificationState "saved"}} {{action (perform update) }}>
+      <button class="cs-version-control-footer--button" disabled={{eq modificationState "saved"}} {{action (perform update) }}>
         Update
       </button>
+      {{#if (is-component (concat modelType '-action-buttons'))}}
+        {{#with (concat modelType '-action-buttons') as |buttonsComponent|}}
```

Comment:
If you move the `with` so it's outside the `if`, you don't need to say `(concat modelType '-action-buttons')` twice.

Reply:
Good catch, fixed this in https://github.com/cardstack/cardstack/pull/52 .